products/motorola.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Motorola Mobility
+permalink: /motorola
+category: device
+iconSlug: motorola
+releasePolicyLink: https://motorola-global-en-roe.custhelp.com/app/software-upgrade/
+activeSupportColumn: true
```

Comment:
`activeSupportColumn: true` means `support` must be explicitly set for all release cycles.

This is required by : https://github.com/endoflife-date/endoflife.date/blob/9a11b2ebe9e14daabaae8238743748a54df9ce96/_auto/validate.py#L55